nan_callbacks.h

```diff
@@ -101,7 +101,7 @@ static const int kIndexPropertyQueryIndex =      5;
 static const int kIndexPropertyFieldCount =      6;
 
 template<typename T, typename P>
-P *GetWrapper(T needle) {
+static inline P *GetWrapper(T needle) {
```

Comment:
> SetInternalFieldPointer only accepts data pointers, not function pointers.

`object->SetAlignedPointerInInternalField(index, reinterpret_cast<void*>(callback))`?  Or is there some subtlety here I'm misunderstanding?